inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/AgentBaseTestsHelper.java

```diff
@@ -104,6 +104,8 @@ private DataConfig getDataConfig(int taskId, String pattern, boolean retry, Long
         fileTaskConfig.setRetry(retry);
         fileTaskConfig.setStartTime(startTime);
         fileTaskConfig.setEndTime(endTime);
+        fileTaskConfig.setDataContentStyle("mix");
+        fileTaskConfig.setDataSeparator("124");
```

Comment:
It is recommended to use macros or add a comment for '124' and 'mix'

Reply:
get